src/app/screens/Receive/index.tsx

```diff
@@ -100,6 +100,13 @@ function Receive() {
       });
   }
 
+  function receiveAnotherPayment() {
+    setPaid(false);
+    setPollingForPayment(false);
+    setInvoice(undefined);
```

Comment:
Using undefined is not a good practice. You can change the invoice type to `{paymentRequest: ...} | null` and use `null` here.